cvat/apps/engine/default_settings.py

```diff
@@ -24,3 +24,8 @@
 """
 Sets the frequency of checking the readiness of the chunk
 """
+
+CVAT_CHUNK_LOCK_TIMEOUT = 50
```

Comment:
Lock timeout is lock ttl, but here it is a timeout for lock acquisition.
I also think that you need to change the default value of `blocking_timeout` inside `get_rq_lock_for_job` because IMHO it is a bad practice to allow endless waiting for a lock at least by default.